framework/include/utils/MooseUtils.h

```diff
@@ -113,6 +113,11 @@ namespace MooseUtils
   std::string shortName(const std::string & name);
 
   /**
+   * Function for string the information before the final / in a parser block
```

Comment:
So this method was in Action and now it's here. If you move it to here the comment should be changed to reflect that this is a utility method. It doesn't have anything to do with the Parser or Action system anymore.